dask_cuda/cli.py

```diff
@@ -5,21 +5,28 @@
 import click
 from tornado.ioloop import IOLoop, TimeoutError
 
-from dask import config
+from dask import config as dask_config
+from distributed import Client
 from distributed.cli.utils import install_signal_handlers
 from distributed.preloading import validate_preload_argv
 from distributed.security import Security
 from distributed.utils import import_term
 
-from ..cuda_worker import CUDAWorker
+from .cuda_worker import CUDAWorker
+from .utils import print_cluster_config
 
 logger = logging.getLogger(__name__)
 
 
 pem_file_option_type = click.Path(exists=True, resolve_path=True)
 
 
-@click.command(context_settings=dict(ignore_unknown_options=True))
+@click.group
```

Comment:
Not sure if all `cuda.commands` need to reside in the same file as the initial definition to get picked up correctly? Made a few failed attempts to keep the initial structure of the CLI modules before settling on having them all in one shared module, which is similar to what dask-ctl does for its grouped `cluster`/`discovery` commands.

Assuming this is the only way to organize to the CLI commands, we could probably consolidate things up a bit by reusing shared click arguments/options.